pipelines/deployer/deployer.yaml

```
@@ -639,6 +639,39 @@ jobs:
     config: *unpack_release
     params:
       CLUSTER_PUBLIC_KEY: ((ci-system-gpg-public))
+  - task: scale-autoscaler-down
+    image: task-toolbox
+    timeout: 10s
+    params:
+      CLUSTER_NAME: ((cluster-name))
+      ACCOUNT_ROLE_ARN: ((account-role-arn))
+    config:
+      platform: linux
+      run:
+        path: /bin/bash
+        args:
+        - -euo
+        - pipefail
+        - -c
+        - |
+          echo "assuming aws deployer role..."
+          AWS_CREDS="$(aws-assume-role $ACCOUNT_ROLE_ARN)"
+          echo "Looking for existing EKS cluster"
+          CLUSTER_COUNT=$(aws eks list-clusters | jq '.clusters[] | select (. == "$CLUSTER_NAME") | 1' | wc -l)
+          if [ "$CLUSTER_COUNT" == "1" ]; then
+              echo "found it, fetching kubeconfig from aws..."
+              aws eks update-kubeconfig \
+                --name "${CLUSTER_NAME}" \
+                --kubeconfig ./kubeconfig
+              echo "done, looking for deployments"
+              export KUBECONFIG=$(pwd)/kubeconfig
+              DEPLOYMENT_COUNT=$(kubectl --namespace gsp-system get deployment -o json | jq '.items[] | select (.metadata.name == "gsp-aws-cluster-autoscaler") | 1' | wc -l)
+              if [ "$DEPLOYMENT_COUNT" == "1" ]; then
```

```suggestion
              if [ "$DEPLOYMENT_COUNT" -gt "0" ]; then
```
This feels a bit more defensive in case $DEPLOYMENT_COUNT is ever more than 1 for some reason? I can't think how it would happen though.